packages/filesystem/src/browser/file-tree/file-tree.ts

```diff
@@ -8,8 +8,9 @@
 import { injectable, inject } from "inversify";
 import URI from '@theia/core/lib/common/uri';
 import { ITreeNode, ICompositeTreeNode, ISelectableTreeNode, IExpandableTreeNode, Tree } from "@theia/core/lib/browser";
-import { FileSystem, FileStat, UriSelection } from "../../common";
+import { FileSystem, FileStat } from "../../common";
 import { LabelProvider } from "@theia/core/lib/browser/label-provider";
+import { UriSelection } from '@theia/core/lib/common//selection';
```

Comment:
`//selection` -> `/selection`

We should reexport `selection` in `@theia/core/lib/common/index.ts` to import from common directly.